scripts/pce-fw-builder.sh

```diff
@@ -1,9 +1,29 @@
-#!/bin/bash
+#!/bin/bash -x
 
 echo $*
 
 cd /home/coreboot/coreboot
-make distclean
-cp configs/pcengines_$1.config .config
-make -j$(nproc)
+
+if [ $# == 1 ];then
+    echo "Build coreboot for $1"
+    make distclean
+    cp configs/pcengines_$1.config .config
+    make -j$(nproc)
+elif [ $# == 2 ]; then
+    echo "Build custom coreboot for $1"
+    if [ "$2" == "nodistclean" ]; then
+        make -j$(nproc)
+    else
+        echo "distclean"
+        make distclean
+        echo "copy $1 config"
+        cp configs/pcengines_$1.config .config
+        # remove platform version
+        shift
+        make -j$(nproc) $1
```

Comment:
For now I don't know how it will be utilized, because there are no `menuconfig-params` specified. However I have doubts. If $1 will be platform name then `make -j8 apu2` will not be recognized as proper make command.